### PR TITLE
Bug 1163169 - Blocking home events on Logo handler r=alive

### PR DESCRIPTION
**Pull request by albertopq**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=75da22c1c384700742809520a38f703f4049292e

File: apps/system/js/init_logo_handler.js [hunk 1]
```diff
@@ -92,13 +92,21 @@ var InitLogoHandler = {
...
+    window.addEventListener('holdhome', this);
+    window.addEventListener('home', this);
     this.logoLoader = logoLoader;
     logoLoader.onnotfound = this._removeCarrierPowerOn.bind(this);
     logoLoader.onload = this._appendCarrierPowerOn.bind(this);
     Service.request('registerHierarchy', this);
   },
 
-  handleEvent: function ilh_handleEvent() {
+  handleEvent: function ilh_handleEvent(evt) {
```
Comment (alivedise) [inline apps/system/js/init_logo_handler.js]: We should check if we are active here...?

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=bcb0f376b528bdb1d6d6df51ba37a2ecb2524f3e

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2e30b8a03c160f09d6131bad00843ea4fba02ef3

**Comment by autolander** [comment]
http://docs.taskcluster.net/tools/task-graph-inspector/#VvpcvSWEQH-EEAyiRblNWQ

The pull request failed to pass integration tests. It could not be landed, please try again.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2e30b8a03c160f09d6131bad00843ea4fba02ef3

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4738019c463200195947639d3db035f39e46ab65

**Pull request merged**